binding.gyp

```diff
@@ -0,0 +1,24 @@
+{
+  "targets": [
+    {
+      "target_name": "drivelist",
+      "include_dirs" : [
+        "<!(node -e \"require('nan')\")",
+        "."
+      ],
+      "sources": [
+        "src/drivelist.cc"
+      ],
+      "conditions": [
+        [ 'OS=="win"', {
+          "sources": [
+            "src/windows/com.cc",
+            "src/windows/scanner.cc",
+            "src/windows/volume.cc",
+            "src/windows/wmi.cc"
+          ]
```

Comment:
So I've been reading on that weird pragma I included there (from the Windows docs), and it seems that its basically equivalent to listing the library here (it tells the compiler while scanning the source code). I moved it to `binding.gyp`, since that is easier to understand.